modules/local/get_chrom_sizes.nf

```diff
@@ -0,0 +1,39 @@
+// Import generic module functions
+include { initOptions; saveFiles } from './common/functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+/*
+ * Get chromosome sizes from a fasta file
+ */
+process GET_CHROM_SIZES {
+    tag "$fasta"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:"genome", publish_id:'') }
+
+    conda (params.enable_conda ? "bioconda::samtools=1.13" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/samtools:1.13--h8c37831_0"
+    } else {
+        container "quay.io/biocontainers/samtools:1.13--h8c37831_0"
+    }
+
+    input:
+    path fasta
+
+    output:
+    path '*.sizes'      , emit: sizes
+    path '*.fai'        , emit: fai
+    path "*.version.txt", emit: version
+
+    script:
+    def software = 'samtools'
+    """
+    samtools faidx $fasta
+    cut -f 1,2 ${fasta}.fai > ${fasta}.sizes
+    echo \$(samtools --version 2>&1) | sed 's/^.*samtools //; s/Using.*\$//' > ${software}.version.txt
```

Comment:
I think you can make it as a nf-core/modules, it's an easy command, but could be worth it to share with others

Reply:
Ok I will get this submitted